packages/admin/cms-admin/src/builds/StartBuildsDialog.tsx

```diff
@@ -0,0 +1,100 @@
+import { gql, useMutation, useQuery } from "@apollo/client";
+import { CancelButton, LocalErrorScopeApolloContext } from "@comet/admin";
+import { Alert, Box, Button, CircularProgress, Dialog, DialogActions, DialogContent, DialogTitle } from "@mui/material";
+import { DataGrid, GridSelectionModel } from "@mui/x-data-grid";
+import * as React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { GQLBuildTemplatesQuery, GQLCreateBuildsMutation, GQLCreateBuildsMutationVariables, namedOperations } from "../graphql.generated";
+
+const buildTemplatesQuery = gql`
+    query BuildTemplates {
+        buildTemplates {
+            id
+            name
+        }
+    }
+`;
+
+const createBuildsMutation = gql`
+    mutation CreateBuilds($input: CreateBuildsInput!) {
+        createBuilds(input: $input)
+    }
+`;
+
+type StartBuildsDialogProps = {
+    open: boolean;
+    onClose: () => void;
+};
+
+export function StartBuildsDialog(props: StartBuildsDialogProps) {
+    const { open, onClose } = props;
+
+    const intl = useIntl();
+
+    const { data } = useQuery<GQLBuildTemplatesQuery>(buildTemplatesQuery, {
+        context: LocalErrorScopeApolloContext,
+    });
+    const [startBuilds, { loading }] = useMutation<GQLCreateBuildsMutation, GQLCreateBuildsMutationVariables>(createBuildsMutation, {
+        refetchQueries: [namedOperations.Query.Builds],
+    });
+
+    const [selectionModel, setSelectionModel] = React.useState<GridSelectionModel>([]);
+
+    const rows = data?.buildTemplates ?? [];
+
+    return (
+        <Dialog open={open} onClose={onClose}>
+            <DialogTitle>
+                <FormattedMessage id="comet.pages.publisher.startBuildsDialog.header" defaultMessage="Start manual builds" />
+            </DialogTitle>
+            <DialogContent>
+                <Alert severity="info">
+                    <FormattedMessage
+                        id="comet.pages.publisher.startBuildsDialog.hint"
+                        defaultMessage="The more builds you start, the longer it takes until they are completed. If you need to rebuild manually, select only the scopes required."
+                    />
+                </Alert>
+                <Box sx={{ height: 400, width: "100%" }}>
```

Comment:
Is it possible to have an `auto` height?

Reply:
https://mui.com/x/react-data-grid/layout/#auto-height

Reply:
auto-height must not be used with autoPageSize. I've had a quick talk with @jamesricky: we decided to go for a fixed page size and auto-height without any surrounding box